src/Mod/BIM/bimcommands/BimArchUtils.py

```diff
@@ -366,9 +366,9 @@ class Arch_IfcSpreadsheet:
 
     def GetResources(self):
         return {'Pixmap': 'Arch_Schedule',
-                'MenuText': QT_TRANSLATE_NOOP("Arch_IfcSpreadsheet","Create IFC spreadsheet..."),
+                'MenuText': QT_TRANSLATE_NOOP("Arch_IfcSpreadsheet","Create IFC Spreadsheet…"),
```

Comment:
```suggestion
                'MenuText': QT_TRANSLATE_NOOP("Arch_IfcSpreadsheet","New IFC Spreadsheet…"),
```

I feel we have always new instead of create?